src/components/PickupBox/PickupBox.module.css

```diff
@@ -131,3 +131,60 @@
 .image-corners {
   border-radius: 5%;
 }
+
+.image-functionality-wrapper {
+  display: flex;
+  width: 100%;
+}
+
+.image-functionality {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.approval-button {
+  float:left;
+  margin: 2% 2% 0% 0%;
+}
+
+.rejection-button {
+  float:left;
+  margin: 2% 2% 0% 0%;
+}
```

Comment:
If you think these CSS selectors should always have the same styling, can we merge them into something like:
.approval-button,
.rejection-button {
  float: left;
  margin: 2% 2% 0% 0%;
}